### ISSUE TITLE
Learning Rate with Adam Updater

### ISSUE THREAD

**Issue by Dathiou** [deeplearning4j/deeplearning4j#4064]
#### Issue Description

When creating my network conf, I did this:

```
					.learningRate(0.07)
					.updater(new Adam(0.002,0.9,0.999,1e-8))
```
And while looking at the UI (by clicking on a layer and scrolling down) I see:
![image](https://user-images.githubusercontent.com/7745592/30399190-b7702bda-9887-11e7-98ac-4ca8df271f04.png)

How is the 0.02 used in this case? 

I also tried this:
```

					.updater(new Adam(0.002,0.9,0.999,1e-8))
```
and the UI is showing a learning rate of 1E-1 (default value). which again is not the value of the adam updater. 

Are these two values interfering with each other? is the UI not showing the right information? 

#### Version Information

Please indicate relevant versions, including, if relevant:
0.9.1

**Comment by AlexDBlack** [comment]
.learningRate(x) method doesn't exist anymore, as of yesterday:
https://github.com/deeplearning4j/deeplearning4j/pull/3985

Previously the intent was you would use one or the other, not both.

See also: https://github.com/deeplearning4j/deeplearning4j/issues/3892

**Comment by Dathiou** [comment]
Thank you. Will it be available starting 0.9.2? How can I make sure the model is not using the default learning rate of 1e-1?

**Comment by AlexDBlack** [comment]
The safest option: .learningRate(x).updater(new Adam(x,....)) on each layer.
That should guarantee that the layer gets learning rate x regardless of any other (inc. global) settings.
